File: src/test/java/org/demo/projections/BankAccountProjectionsEvents.java
package org.demo.projections;

import org.demo.domain.BankAccountEvents;

import java.math.BigDecimal;

public class BankAccountProjectionsEvents {
	public static BankAccount emptyBankAccount() {
		return new BankAccount(BankAccountEvents.ACCOUNT_ID, BankAccountEvents.ACCOUNT_NUMBER, BigDecimal.ZERO);
	}
}

File: src/test/java/org/demo/projections/BankAccountProjectionsEvents.kt
package org.demo.projections

import org.demo.domain.BankAccountEvents
import java.math.BigDecimal

object BankAccountProjectionsEvents {
    fun emptyBankAccount(): BankAccount {
        return BankAccount(BankAccountEvents.ACCOUNT_ID, BankAccountEvents.ACCOUNT_NUMBER, BigDecimal.ZERO)
    }
}
